crates/node/src/storage/database/postgres.rs

```diff
@@ -535,6 +535,14 @@ impl Database {
         Ok(())
     }
 
+    pub async fn get_acl_whitelist(&self) -> Result<Vec<entity::PublicKey>> {
+        let dbkey_set: Vec<entity::PublicKey> = sqlx::query("SELECT 1 FROM acl_whitelist")
```

Comment:
Did you test this? This query returns literal `1` for each row that exists in the DB. I'd expect this to yield error when trying to parse as public key?

Reply:
Yes, I copy and paste from another query. I thought the 1 was interpreted as a prepared statement, the first return field. I didn't test the read in fact because I start only one time.
I correct and to a read test.

Reply:
I've to add a test too, when the public key parse fail.

Reply:
I've corrected the query and added the test.